docs/man/man1/manta-adm.md

```diff
@@ -794,6 +794,12 @@ options described above, plus:
   When upgrading a zone, always provision a new zone and deprovision the
   previous one, rather than reprovisioning the existing one.
 
+`--skip-verify-channel`
```

Comment:
```suggestion
`--skip-verify-remote`
```

Reply:
Thanks!

Reply:
.. and I missed a "not" in the line immediately below it, fixing that too.